services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -1382,50 +1296,44 @@
                     }
                   },
                   {
-                    "id": "cam-8",
+                    "id": "accessMeasure_oversightMethodFrequency",
                     "type": "radio",
                     "validation": "radio",
                     "props": {
                       "label": "C2.V.8 Frequency of oversight methods",
                       "hint": "What is the frequency of oversight methods? Select one or add free text if you select \"Other\"",
                       "choices": [
                         {
-                          "name": "cam-8-o1",
-                          "type": "choice",
+                          "name": "option1",
                           "label": "At procurement"
                         },
                         {
-                          "name": "cam-8-o2",
-                          "type": "choice",
+                          "name": "option2",
                           "label": "Monthly"
                         },
                         {
-                          "name": "cam-8-o3",
-                          "type": "choice",
+                          "name": "option3",
                           "label": "Quarterly"
                         },
                         {
-                          "name": "cam-8-o4",
-                          "type": "choice",
+                          "name": "option4",
                           "label": "Annually"
                         },
                         {
-                          "name": "cam-8-o5",
-                          "type": "choice",
+                          "name": "option5",
                           "label": "Less than annually"
                         },
                         {
-                          "name": "cam-8-o6",
-                          "type": "choice",
+                          "name": "option6",
                           "label": "Other, specify",
                           "children": [
                             {
-                              "id": "cam-8-o6-text",
+                              "id": "accessMeasure_oversightMethodFrequency-otherText",
                               "type": "textarea",
                               "validation": {
                                 "type": "radio",
                                 "nested": true,
-                                "parentFieldName": "cam-6",
+                                "parentFieldName": "population",
```

Comment:
double checking: should this be `population` or should it be `accessMeasure_oversightMethodFrequency`

Reply:
Looks like it should be accessMeasure_oversightMethodFrequency

Reply:
good catch